### DISCUSSION TITLE
Login GitHub Copilot

### DISCUSSION THREAD

**Discussion by germanolira** [community/community#36581] [Copilot Conversations]
Today I opened vscode and I couldn't log in to github copilot, it only opens a web page and doesn't log in to vscode, it opens my profile, everything is ok with the payment, it just can't redirect me to the web login page

![Captura de Tela 2022-10-18 às 15 03 20](https://user-images.githubusercontent.com/25235170/196509643-70e96517-f15e-4cdb-99f3-ecf32a349654.png)

**Comment by BLKKKBVSIK** [comment]
Hi @germanolira,

Thanks for being a part of the GitHub Community! We're glad you're here.
It may be related to the file stored on your computer generated by VSC.

A good way would be to clean your tmp files on your computer and to reset user preferences files.

**Comment by davecheney** [comment]
@germanolira which version of VSCode are you using?

**Reply by germanolira** [reply]
Today when I opened vscode it showed up the modal to log into github copilot, but the version I'm using today is this

Version: 1.69.2

**Reply by davecheney** [reply]
Could you please check and confirm you are not using VScodium or OSS Code. 

<img width="475" alt="Screen Shot 2021-11-30 at 8 03 57 am" src="https://user-images.githubusercontent.com/7171/143942796-0c2b6d34-e0f4-46fe-b349-f26087b2c5ed.png">
Please try signing out, copilot will prompt you to sign back in.

**Comment by ossanna16** [comment]
Hey there @germanolira and thank you for asking a good question! I'm glad you found our community 🙂
